src/app/experts/enter-response/enter-response.component.ts

```diff
@@ -57,7 +57,9 @@ export class EnterResponseComponent implements OnInit {
             duration: 3000,
           });
           this.response.body = null;
-          this.router.navigate(['/view-question']);
+          this.router.navigate(['/questions']); //Currently routes to settings because it does
+          //not refresh the page if something is submitted, so person will have to navigate
+          //back to see changes.
```

Comment:
Let's make the existing newResponse.emit(response) functionality work by asking the backend for an updated list of responses, instead of refreshing via navigation.  It looks like the event is being emitted here but nothing is done with it.